src/index.ts

```diff
@@ -459,9 +474,37 @@ const createTable = (tableName: string): TE.TaskEither<Error, void> =>
     (err) => new Error(`Could not create table: ${err}`)
   );
 
+const messagesCollectionIndexingPolicy = {
```

Comment:
Create a new type extending the `IndexingPolicy` type to handle composite indexes definitions
```suggestion
interface CompositeIndexingPolicy extends IndexingPolicy {
  compositeIndexes: ReadonlyArray<ReadonlyArray<IndexPath>>
}

enum IndexOrder {
  ASC = "ascending",
  DESC = "descending"
}

interface IndexPath {
  order: IndexOrder,
  path: string
}

const messagesCollectionIndexingPolicy: CompositeIndexingPolicy = {
```

Reply:
I moved those new types in a new file in https://github.com/pagopa/io-fixtures/pull/15/commits/2c1b30dc071277efe1b549d4e13477497b16675a. 
Thank you for the review.